<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZincWindowWindow</class>
 <widget class="QDockWidget" name="ZincWindowWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zinc</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="layout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <action name="actionAxes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Axes</string>
   </property>
   <property name="toolTip">
    <string>Axes</string>
   </property>
   <property name="statusTip">
    <string>Show/hide axes</string>
   </property>
  </action>
  <action name="actionPoints">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Points</string>
   </property>
   <property name="toolTip">
    <string>Points</string>
   </property>
   <property name="statusTip">
    <string>Show/hide points</string>
   </property>
  </action>
  <action name="actionLines">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Lines</string>
   </property>
   <property name="toolTip">
    <string>Lines</string>
   </property>
   <property name="statusTip">
    <string>Show/hide lines</string>
   </property>
  </action>
  <action name="actionSurfaces">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Surfaces</string>
   </property>
   <property name="toolTip">
    <string>Surfaces</string>
   </property>
   <property name="statusTip">
    <string>Show/hide surfaces</string>
   </property>
  </action>
  <action name="actionIsosurfaces">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Isosurfaces</string>
   </property>
   <property name="toolTip">
    <string>Isosurfaces</string>
   </property>
   <property name="statusTip">
    <string>Show/hide isosurfaces</string>
   </property>
  </action>
  <action name="actionTrilinearCube">
   <property name="text">
    <string>Trilinear Cube</string>
   </property>
   <property name="iconText">
    <string>Trilinear Cube</string>
   </property>
   <property name="toolTip">
    <string>Trilinear Cube</string>
   </property>
   <property name="statusTip">
    <string>Load the trilinear cube mesh</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
